tests/dummy/app/templates/components/cookbook/horizontal-scrolling-table.hbs

```diff
@@ -1,6 +1,11 @@
 {{!-- BEGIN-SNIPPET horizontal-scrolling-table --}}
 {{#light-table table height='65vh' as |t|}}
 
+  {{!--
+    In order for `fa-sort-asc` and `fa-sort-desc` icons to work,
+    you need to have ember-font-awesome installed.
```

Comment:
technically we only need `font-awesome` (e.g. via a CDN), do we care to highlight the difference?